src/auth/integration-tests/src/lib.rs

```diff
@@ -457,12 +456,18 @@ async fn get_secret_with_mds_creds(
     Ok(payload)
 }
 
+#[cfg(google_cloud_unstable_id_token)]
+use unstable::generate_id_token;
+
```

Comment:
Why introduce `unstable::generate_id_token()` and then immediately define a function with the same name?

Reply:
it is a way to have integration tests for impersonated::idtokens. But can be removed 🤷